packages/jellyfish-api-core/__tests__/category/account/listBurnHistory.test.ts

```diff
@@ -0,0 +1,261 @@
+    expect(history[1].blockHash.length).toStrictEqual(64)
+    expect(history[1].type).toStrictEqual('CreateToken')
+    expect(history[1].txn).toStrictEqual(1)
+    expect(typeof history[1].txid).toStrictEqual('string')
+    expect(history[1].txid.length).toStrictEqual(64)
+    expect(typeof history[1].amounts).toStrictEqual('object')
+    expect(history[1].amounts.length).toStrictEqual(1)
+    expect(history[1].amounts[0]).toStrictEqual('1.00000000@DFI')
+  })
+
+  it('third transaction should be send token to burn address', async () => {
+    expect(history[2].owner).toStrictEqual(burnAddress)
+    expect(history[2].type).toStrictEqual('AccountToAccount')
+    expect(typeof history[2].amounts).toStrictEqual('object')
+    expect(history[2].amounts.length).toStrictEqual(1)
+    expect(history[2].amounts[0]).toStrictEqual('100.00000000@GOLD')
+  })
+
+  it('fourth transaction should be utxostoaccount burn', async () => {
+    expect(history[3].owner).toStrictEqual(burnAddress)
+    expect(history[3].type).toStrictEqual('UtxosToAccount')
+    expect(typeof history[3].amounts).toStrictEqual('object')
+    expect(history[3].amounts.length).toStrictEqual(1)
+    expect(history[3].amounts[0]).toStrictEqual('1.00000000@DFI')
+  })
+
+  it('fifth transaction should be send to burn address with accountToAccount', async () => {
+    expect(history[4].owner).toStrictEqual(burnAddress)
+    expect(history[4].type).toStrictEqual('AccountToAccount')
+    expect(typeof history[4].amounts).toStrictEqual('object')
+    expect(history[4].amounts.length).toStrictEqual(1)
+    expect(history[4].amounts[0]).toStrictEqual('1.00000000@DFI')
+  })
+
+  it('sixth transaction should be send to burn address with accounttoutxos', async () => {
+    expect(history[5].owner).toStrictEqual(burnAddress)
+    expect(history[5].type).toStrictEqual('None')
+    expect(typeof history[5].amounts).toStrictEqual('object')
+    expect(history[5].amounts.length).toStrictEqual(1)
+    expect(history[5].amounts[0]).toStrictEqual('2.00000000@DFI')
+  })
+
+  it('last transaction should be send utxo to burn address', async () => {
+    expect(history[6].owner).toStrictEqual(burnAddress)
+    expect(history[6].type).toStrictEqual('None')
+    expect(typeof history[6].amounts).toStrictEqual('object')
+    expect(history[6].amounts.length).toStrictEqual(1)
+    expect(history[6].amounts[0]).toStrictEqual('10.00000000@DFI')
+  })
+
+  it('should listBurnHistory with filter on txtype None', async () => {
+    const history = await client.account.listBurnHistory({
+      txtype: DfTxType.NONE
+    })
+    expect(history.every(({ type }) => type === 'None')).toBeTruthy()
+  })
+
+  it('should listBurnHistory with filter on txtype UtxosToAccount', async () => {
+    const history = await client.account.listBurnHistory({
+      txtype: DfTxType.UTXOS_TO_ACCOUNT
+    })
+    expect(history.every(({ type }) => type === 'UtxosToAccount')).toBeTruthy()
```

Comment:
For those scenarios where the length of the history is 1, write something like this. You do not need to use `every`
```

    expect(history.length).toStrictEqual(1)
    expect(history[0].type).toStrictEqual('UtxosToAccount')
```